### PR TITLE
SAVAMC2-85 Import fixes

### PR DESCRIPTION
**Pull request by jkufro**
# Description
Jira Ticket: SAVAMC2-85

Write out a concise summary of this pull request and what it addresses.

# (Feature) Demo/Screenshots
Insert demo video or photos for a new or updated feature or capability.

# (Bugfix) How to Replicate
Insert steps for how to replicate the bug this PR addresses.

# (Bugfix) Solution
Insert description for the solution this PR implements to address the bug.

# Important Changes
Please list important files (meaning substantial or integral to the PR) along with a list of the general changes that should be highlighted for reviewers.

`example_file.js`
- Example change (ex: refactored import function)

# Testing
This fix was tested on the following browsers (submitter must check all those that apply):
* [ ] Chrome
* [ ] Firefox
* [ ] Safari
* [ ] IE11

Please describe the tests needed to verify your changes. Provide instructions for reproducing these tests. List any relevant details for your test configuration.

### CODE REVIEW

File: app/controllers/import_controller.rb [hunk 1]
```diff
@@ -15,16 +15,16 @@ def index
 
   def download_guidance
     send_file(
-      "#{Rails.root}/public/Sara%20Alert%20Import%20Format.xlsx",
-      filename: 'Sara%20Alert%20Import%20Format.xlsx',
+      "#{Rails.root}/public/VACCS%20Import%20Format.xlsx",
+      filename: 'VACCS%20Import%20Format.xlsx',
```
Comment (jkufro) [inline app/controllers/import_controller.rb:19]: @ChristineDuong @kevstoy The interesting thing is that Sara Alert is pointing the download link to the version on GitHub from the master branch. I mirrored the same, so the way this is right now, the link will not work until this is merged into master. 

This is the same file attached here
[VACCS Import Format.xlsx](https://github.com/SaraAlertVaccine/SaraAlert/files/5638413/VACCS.Import.Format.xlsx)

Comment (ChristineDuong) [reply app/controllers/import_controller.rb:19]: [VACCS.Import.Format.xlsx](https://github.com/SaraAlertVaccine/SaraAlert/files/5638812/VACCS.Import.Format.xlsx)

Made small, minor changes:
* Unbolded "Last Date of Exposure" in Format Guidance spreadsheet.
* Added light blue squares to gender identity columns to indicate it was part of the enrollment process in Template and Format spreadsheet

Let me know if that makes sense.

**Review by ChristineDuong — CHANGES_REQUESTED** [review]
Screen Shot 2020-12-03 at 3 33 53 PM
This might be a @kevstoy question; do we want to change this import review to not include monitoring information, etc.

**Comment by jkufro** [comment]
> Screen Shot 2020-12-03 at 3 33 53 PM
> This might be a @kevstoy question; do we want to change this import review to not include monitoring information, etc.

@ChristineDuong @kevstoy IMO this would be something to remove since we already removed the same info from the patient show page. I can remove this tomorrow as long as @kevstoy does not object.

**Review by ChristineDuong — APPROVED** [review]

**Review by kevstoy — APPROVED** [review]

**Pull request merged**